### PR TITLE
Join us page for DWoC

### PR DESCRIPTION
**Pull request by vedikaag99**
## Issue that this pull request solves

 Closes: #277 (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

https://user-images.githubusercontent.com/67699840/106389084-8e4ea700-6407-11eb-959f-3345a1db752c.mov

## Other information

Any other information that is important to this pull request
no

### CODE REVIEW

File: frontend/src/components/home/join_us/join_us_form/form.css [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        margin-bottom: 1rem;
+    }
+    
+    .joinusform-input1 {
+        margin-bottom: 1.5rem;
+    }
+    
+    input[type="date"]:not(:valid):before {
+        content: attr(placeholder);
+        /* style it like it is a real placeholder */
```
Comment (Surajbokde) [inline frontend/src/components/home/join_us/join_us_form/form.css:188]: remove this comment

File: frontend/src/components/home/join_us/join_us_form/form.css [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    .submit-btn-text {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      font-size: x-small;
+      color: black;
+    }
+  }
+  
+@media only screen and (max-width: 640px) {
```
Comment (Surajbokde) [inline frontend/src/components/home/join_us/join_us_form/form.css:258]: i think there must be 1 space before @media .

File: frontend/src/components/home/join_us/join_us_form/form.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import "./form.css";
+import { Button2 } from "../../../util/button/button";
+const JoinUsForm = () => {
+    return (
+        <div className="joinusform-section ">
+      <div className="joinusform-image child1">
+        <img src="./images/joinus.jpg" alt="" />
```
Comment (Surajbokde) [inline frontend/src/components/home/join_us/join_us_form/form.js:8]: reduce 1 extra space

File: frontend/src/components/home/join_us/join_us_form/form.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
+import React from "react";
+import "./form.css";
+import { Button2 } from "../../../util/button/button";
+const JoinUsForm = () => {
+    return (
+        <div className="joinusform-section ">
```
Comment (Surajbokde) [inline frontend/src/components/home/join_us/join_us_form/form.js:6]: reduce 1 extra space

File: frontend/src/components/home/join_us/join_us_form/form.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+                  placeholder="Linkedin Profile URL"
+                  id="txt_link"
+                  type="text"
+                  Required="required"
+                  name="link"
+                />
+                <i class="fas fa-link"></i>
+              </div>
+            </div>
+
```
Comment (Surajbokde) [inline frontend/src/components/home/join_us/join_us_form/form.js:64]: remove this empty line

**Review by tripathyprateek — APPROVED** [review]
Looks good to me!

**Review by Surajbokde — COMMENTED** [review]
It looks good to me .
remove extra space and comment.

**Comment by vedikaag99** [comment]
@Surajbokde Please check now.

**Review by Surajbokde — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**